recipes/mss/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = "1.1.0" %}
+
+
+package:
+  name: mss
+  version: {{ version }}
+
+source:
+    git_url: https://bitbucket.org/wxmetvis/mss.git
+    git_rev: 1.1.0
+
+build:
+    number: 0
+    script: python setup.py install --record record.txt
+    entry_points:
+      - mss = msui.mss_pyui:main
+
+requirements:
+  build:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl ==7.26
+    - cycler ==0.10.0
+    - execnet ==1.2.0
+    - freetype >=2.5.0
+    - functools32 >=3.2.3.2
+    - geopy >=1.10.0
+    - hdf5 >=1.8.15
+    - jpeg >=8d
```

Comment:
These need to match our [pinned version]( https://github.com/conda-forge/conda-forge.github.io/blob/352aab2876ff1dd4ad0163c67384f29b5e56646d/scripts/pin_the_slow_way.py#L32-L48 ).